helmnet/hybridnet.py

```diff
@@ -220,6 +251,17 @@ def val_dataloader(self):
             num_workers=min([self.hparams.batch_size, 32]),
         )
 
+    def test_dataloader(self):
+        self.reset_source()
+        testset = get_dataset("datasets/splitted_96/testset.ph")
```

Comment:
Should we pass this path to the `IterativeSolver` from the `settings`?  I think that different people may use different datasets, possibly in the same project

Reply:
Sounds great, will include in next commit.